src/chainstate/stacks/boot/pox_2_tests.rs

```diff
@@ -417,3 +417,324 @@ fn test_simple_pox_lockup_transition_pox_2() {
         "Bob tx1 should have resulted in a runtime error"
     );
 }
+
```

Comment:
This is the old version of the test?

Reply:
I'm not sure what this comment refers to -- the `test_simple_pox_lockup_transition_pox_2` test is, I think, unchanged in this  PR. The new tests were based on the loop-rolled version of that test, but I've since unrolled them.